pom.xml

```diff
@@ -79,38 +81,61 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-common</artifactId>
+      <version>4.1.15.Final</version>
```

Comment:
Why don't you set this in `vertx-dependencies`? Otherwise you have the versions defined at 2 locations, and potentially diverging for the stack.

Reply:
it's temporary, I will remove this when the PR will be merged, see the note in comments